manifests/v1beta1/installs/katib-openshift/kustomization.yml

```diff
@@ -0,0 +1,71 @@
+# This kustomization copies the `katib-standalone` one with following exclusions:
```

Comment:
Let's also update the release script: https://github.com/kubeflow/katib/blob/master/scripts/v1beta1/release.sh#L77.
Then, we will update the `katib-openshift` install images for the release branch also.